frontend/game/hand.js

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+define(function(require) {
+    var helper = require('./helper.js');
+    return {
+        setHand(handState) {
```

Comment:
Is this called pretty often or only once?
Because it feels like a slow browser could be showing the hand to disappear and reappear... (maybe I'm just saying silly things)

Reply:
Check out https://github.com/kenan-rhoton/card-game/pull/49/files/7ecddddebdc458687aac0a8f7ccc29a4bcf1f806#diff-3075d78b5d6ef0ac1276cb712385903fR18

Reply:
So it only reloads it when the number of cards changed?
Will need change if we create any new rule that makes you "draw" new cards, but right new seems like a nice solution.